app/models/user.rb

```diff
@@ -27,6 +27,10 @@
 
 class User < ActiveRecord::Base
 
+  default_scope { where(deleted_at: nil) }
+
+  scope :including_deleted, -> { unscoped }
```

Comment:
Consider changing this to `where deleted_at not nil`, because this would override all scopes when chained.